digits/config/store_option.py

```diff
@@ -15,19 +18,34 @@ def prompt_title(cls):
         return 'Model Store URL list'
 
     @classmethod
+    def prompt_message(cls):
+        return 'What Model Stores should DIGITS look for models?'
+
+    @classmethod
     def visibility(cls):
-        return config_option.Visibility.HIDDEN
+        return config_option.Visibility.DEFAULT
+
+    @classmethod
+    def validate(cls, value):
+        valid_url_list = list()
+        if isinstance(value, str):
+            url_list = value.split(',')
+            for url in url_list:
+                if url is not None and urlparse(url).scheme != "" and not os.path.exists(url):
+                    valid_url_list.append(url)
+        return ','.join(valid_url_list)
+
 
     def optional(self):
         return True
 
     def suggestions(self):
-        self.url_list = ['http://localhost/modelstore/']
-        return [prompt.Suggestion(self.url_list, 'H', desc='URL list')]
+        self.url_list = ''
+        return [prompt.Suggestion(self.url_list, 'M', desc='Model Store URL list (comma separated)')]
```

Comment:
Please put the format requirements in `prompt_message()` above. I don't think you actually need any suggestions for now.